src/main/java/com/epam/university/java/core/dmitry_novikov_task1/Task1Impl.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.university.java.core.dmitry_novikov_task1;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Author Dmitry Novikov 20-Sep-20.
+ */
+public class Task1Impl implements Task1{
```

Comment:
There is no need to include the solution into the contribution so could you please remove it from the PR?